### ISSUE TITLE
Unrecognized option: --aspects_parameters=optimize_building_jars=enabled

### ISSUE THREAD

**Issue by ruozhuochen** [bazelbuild/intellij#6649]
### Description of the bug:

We have encountered this error after updating to Bazel plugin 2024.07.30.0.1-api-version-241. Essentially, this option "--aspects_parameters" is not supported by bazel 5.4, which is what we are using at the moment. Is there a way for us to bypass this flag without updating the bazel version ? 

Error: 

```
Command: bazel build --tool_tag=ijwb:IDEA:ultimate "--override_repository=intellij_aspect=/Users/${USER}/Library/Application Support/JetBrains/IntelliJIdea2024.1/plugins/ijwb/aspect" --keep_going --build_event_binary_file=/var/folders/8v/gtdhdlwn4wq30p352jnvzrsw0000gp/T/intellij-bep-5bb15506-2452-4a4a-a902-23854967975d --nobuild_event_binary_file_path_conversion --curses=no --color=yes --progress_in_terminal_title=no --noexperimental_run_validations --aspects=@intellij_aspect//:intellij_info_bundled.bzl%intellij_info_aspect --aspects_parameters=optimize_building_jars=enabled --output_groups=intellij-resolve-java,intellij-resolve-py,intellij-info-generic,intellij-info-java,intellij-info-py -- //evolution-service:evolution_service
ERROR: --aspects_parameters=optimize_building_jars=enabled :: Unrecognized option: --aspects_parameters=optimize_building_jars=enabled
```

### Which category does this issue belong to?

Intellij

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Use bazel 5.4. Install Intellij 2024.1.4 and bazel plugin 2024.07.30.0.1-api-version-241

### Which Intellij IDE are you using? Please provide the specific version.

Intellij 2024.1.4

### What programming languages and tools are you using? Please provide specific versions.

Java

### What Bazel plugin version are you using?

2024.07.30.0.1-api-version-241

**Comment by ruozhuochen** [comment]
Closing ticket. 

Just need to remove the option --noexperimental_allow_top_level_aspects_parameters

**Comment by ruozhuochen** [comment]
Confirmed that I can still see the error.

**Comment by mai93** [comment]
We released a fix in version [2024.07.30.0.2](https://plugins.jetbrains.com/plugin/8609-bazel-for-intellij/versions/stable/591230)
